spikes/xml/claim.xml

```diff
@@ -0,0 +1,237 @@
+            <!--Computer experts-->
+            <!--Consultant medical reports-->
+            <!--Costs judge application fee-->
+            <!--Costs judge preparation award-->
+            <!--DNA testing-->
+            <!--Engineer-->
+            <!--Enquiry agents-->
+            <!--Facial mapping expert-->
+            <!--Financial expert-->
+            <!--Fingerprint expert-->
+            <!--Fire assessor/explosives expert-->
+            <!--Forensic scientists-->
+            <!--Handwriting expert-->
+            <!--Interpreter-->
+            <!--Lip readers-->
+            <!--Medical expert-->
+            <!--Memorandum of conviction fee-->
+            <!--Meteorologist-->
+            <!--Other-->
+            <!--Overnight expenses-->
+            <!--Pathologist-->
+            <!--Photocopying-->
+            <!--Psychiatric reports-->
+            <!--Psychological report-->
+            <!--Surveyor/architect-->
+            <!--Transcripts-->
+            <!--Translator-->
+            <!--Vet report-->
+            <!--Voice recognition-->
+            <disbursement_types>Accident reconstruction report</disbursement_types>
+            <net_amount>9.70</net_amount>
+            <vat_amount>1.94</vat_amount>
+        </disbursement>
+    </disbursements>
+
+    <documents>
+        <!--Zero or more documents - always pdf-->
+        <document>
+            <uuid>345fa455-2526-47d9-9ca0-bcc4afddfa68</uuid>
+            <file_name>Hogdson_-_re_order.pdf</file_name>
+            <!--URI refers to location where document is stored in Amazon S3 bucket-->
+            <uri>documents/000/044/782/Hogdson_-_rep_order.pdf</uri>
+            <size>388031</size>
+        </document>
+    </documents>
+
+
+
+    <messages>
+        <!--Zero or more messages-->
+        <message>
+            <!--UUID of message sender - either caseworker or external user-->
+            <sender_uuid>345fa455-2526-47d9-9ca0-bcc4afddfa68</sender_uuid>
+            <body>Please find enclosed invoice for printhouse.  This was enclosed with our original claim however, we have enclosed it again.  \r
+                As you will see from the invoice the copies are for colour copies.  We do not have facilities in our office to produce colour photos.\r
+                We also enclose an amended more detailed invoice from the Private Investigator.\r                                                                                                                                                                                   +
+                We hope the above is now sufficient to be able to pay these invoices..
+            </body>
+            <created_at>2016-09-29T13:11:43Z</created_at>
+            <!--Optional attached document - UUID refers to a document specified in the document section above-->
+            <document_uuid>345fa455-2526-47d9-9ca0-bcc4afddfa68</document_uuid>
+        </message>
+    </messages>
+
+    <determinations>
```

Comment:
This might be confusing, being a collection of 'determinations' but the elements called 'assessment' or 'redetermination'.
I wonder if we could simplify this or in the worst case create 2 different sections, one for the assessment and another for the 'determinations'?